### PR TITLE
Update Indian tax rates

### PR DESCRIPTION
**Pull request by ashishsharawat**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Update Indian tax rates
| Type?         | improvement
| Category?     | LO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | [EPIC](https://github.com/PrestaShop/PrestaShop/issues/15829) for the rework of the localization packs
| How to test?  | Please indicate to https://cbic-gst.gov.in/gst-goods-services-rates.html or https://www.kotak.com/en/stories-in-focus/gst-rate-slabs.html to verify how rates vary all over INDIA <br>Check if values are good after install

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16000)

### CODE REVIEW

**Comment by Progi1984** [comment]
Poke @LouiseBonnard

**Comment by LouiseBonnard** [comment]
Hi @ashishsharawat, thank you very much for this great PR! Should it be called GST or IGST? I can read both here.

**Comment by ashishsharawat** [comment]
Both can be used whether GST or IGST (IGST is used in billing and taxation For Inter-state supply) I think the online billing should use GST not in IGST so it's better to use GST even PayPal bill receipt printed GST instead of IGST!

File: localization/in.xml [hunk 1]
```diff
@@ -10,21 +10,26 @@
 		<currency name="Indian Rupee" iso_code="INR" iso_code_num="356" sign="₹" blank="1" conversion_rate="60.748205" format="1" decimals="1" />
 	</currencies>
 	<taxes>
-		<tax id="1" name="VAT IN 12.5%" rate="12.5" />
-		<tax id="2" name="VAT IN 4%" rate="4" />
-		<tax id="3" name="VAT IN 1%" rate="1" />
-
-		<taxRulesGroup name="IN Standard Rate (12.5%)">
+		<tax id="1" name="IGST 5%" rate="5" />
```
Comment (LouiseBonnard) [inline localization/in.xml]:
```suggestion
		<tax id="1" name="GST 5%" rate="5" />
```

File: localization/in.xml [hunk 2]
```diff
@@ -10,21 +10,26 @@
...
+		<tax id="2" name="IGST 12%" rate="12" />
```
Comment (LouiseBonnard) [inline localization/in.xml]:
```suggestion
		<tax id="2" name="GST 12%" rate="12" />
```

File: localization/in.xml [hunk 3]
```diff
@@ -10,21 +10,26 @@
...
+		<tax id="3" name="IGST 18%" rate="18" />
```
Comment (LouiseBonnard) [inline localization/in.xml]:
```suggestion
		<tax id="3" name="GST 18%" rate="18" />
```

File: localization/in.xml [hunk 4]
```diff
@@ -10,21 +10,26 @@
...
+    <tax id="4" name="IGST 28%" rate="28" />
```
Comment (LouiseBonnard) [inline localization/in.xml]:
```suggestion
    <tax id="4" name="GST 28%" rate="28" />
```

**Review by ashishsharawat — COMMENTED** [review]
Looks good!

**Review by Progi1984 — APPROVED** [review]

**Comment by LouiseBonnard** [comment]
@Progi1984, I already reviewed it, it's okay for me. ;-)

**Review by LouiseBonnard — APPROVED** [review]

**Pull request merged**